src/main/java/frc/robot/Alignable.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+import java.util.function.BiFunction;
+
+public enum Alignable {
+    AMP(0,0,(x,y)-> new Rotation2d(Math.PI / 2)),
+    SPEAKER(0.46,5.67,(x2,y2)-> new Rotation2d(RobotContainer.shouldFlip() ? 0 : Math.PI).plus(new Rotation2d(Math.atan(y2/x2))));
```

Comment:
Its probably time to rename `shouldFlip` with isRedAliance()

Reply:
done